controllers/manager/monitor.go

```diff
@@ -429,6 +445,27 @@ func ManageStrategy(management CloudManager) bool {
 			_, err := management.GcCreate(client, request)
 			if err != nil {
 				log.Error(err, "Strategy creation failed")
+				// If existing strategy is in applied state delete it so that
+				// new strategy can be created, otherwise, monitor ongoing
+				// strategy.
+				if IsExistingStrategy(err) {
+					strategy_status, _ := management.GcShow(client)
```

Comment:
Can we move the GcShow above to avoid the GcCreate?

Reply:
Done.